src/main/drivers/motor.c

```diff
@@ -44,21 +44,21 @@
 
 #include "motor.h"
 
-static FAST_DATA_ZERO_INIT motorDevice_t *motorDevice;
+static FAST_DATA_ZERO_INIT motorDevice_t motorDevice;
```

Comment:
(for future changes, keep this one)

 it is usually better to rename variable in this case to avoid possible problems during refactoring. 

Another possibility is to use:
static FAST_DATA_ZERO_INIT motorDevice_t motorDeviceSingleton;
static FAST_DATA_ZERO_INIT motorDevice_t * const motorDevice = &motorDeviceSingleton;